CMakeLists.txt

```diff
@@ -186,7 +186,7 @@ set(CMAKE_PREFIX_PATH ${CMAKE_SYSTEM_PREFIX_PATH})
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 	configure_file(org.adi.pkexec.osc.policy.cmakein ${CMAKE_CURRENT_BINARY_DIR}/org.adi.pkexec.osc.policy @ONLY)
 	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/org.adi.pkexec.osc.policy
-		DESTINATION /usr/share/polkit-1/actions/)
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/polkit-1/actions/)
```

Comment:
Did you actually tested this? I'm not really sure, but I think polkit is looking actions in this location, so I'm not sure this will really work...

Reply:
Well, if you choose a different `PREFIX` from `/usr` this will install polkit-1 actions under the new prefix. This is, how `PREFIX` usually works. Otherwise distributions have no means to divert the output to another location for packaging. Don't the CI tests check this? I tested it on a distribution that needs to package the program into a different location from `/usr`. There, it works as intended.

Reply:
Yeah, I know it will install in another location. My point is, will it work?? I'm not so sure it will since polkit is looking for actions in `/usr/share/polkit-1/actions/` (AFAIK)? i don't think CI will test this. To test this, you need to see if you can run osc with sudo rights.

Reply:
> on a distribution that needs to package the program into a different location from /usr

So this looks like a valid thing to me for this change... Please add your `Signed-of` tag to the commit...

Reply:
polkit does not work anymore. This change is not ok.